tests/test_flow/test_persistence.py

```diff
@@ -1378,14 +1383,22 @@ def read(self, path):
 
         class Tracked:
             n_instances_in_memory = 0
```

Comment:
Do we still need this variable since you can use the set to get the count?

Reply:
Yeah, I was wondering about whether to do that. I didn't want to convert `n_instances_in_memory` to a function, but having a property on a class is a little awkward too. (Looks like you can do it like this: https://stackoverflow.com/questions/128573/using-property-on-classmethods)

I don't feel strongly about it either way; what do you all think?